This addition defines NREMOTE, the number of remote mount points.
***************
*** 26,27
   * Machine-independent constants
   */

--- 26,30 -----
   * Machine-independent constants
   */
+ #ifdef REMOTEFS
+ #define	NREMOTE	8		/* number of remote file systems */
+ #endif REMOTEFS
